remote-api/src/main/java/org/alfresco/rest/api/nodes/NodesEntityResource.java

```diff
@@ -65,10 +84,25 @@
 @EntityResource(name="nodes", title = "Nodes")
 public class NodesEntityResource implements
         EntityResourceAction.ReadById<Node>, EntityResourceAction.Delete, EntityResourceAction.Update<Node>,
-        BinaryResourceAction.Read, BinaryResourceAction.Update<Node>, InitializingBean
+        BinaryResourceAction.Read, BinaryResourceAction.Update<Node>, FolderResourceAction.RetrieveFolderSize<Map<String,Object>>, InitializingBean
```

Comment:
I don't see the reason for `FolderResourceAction.RetrieveFolderSize`. Also, this is a Node resource and it should return a `Node` object.